editoast/src/main.rs

```diff
@@ -60,6 +62,26 @@ async fn run() -> Result<(), Box<dyn Error + Send + Sync>> {
     }
 }
 
+fn init_sentry(args: &RunserverArgs) -> Option<ClientInitGuard> {
+    match (args.sentry_dsn.clone(), args.sentry_env.clone()) {
+        (Some(sentry_dsn), Some(sentry_env)) => Some(sentry::init((
+            sentry_dsn,
+            sentry::ClientOptions {
+                release: match env::var("OSRD_GIT_DESCRIBE").ok() {
+                    Some(release) => Some(release.into()),
+                    None => sentry::release_name!(),
+                },
```

Comment:
```suggestion
                release: env::var("OSRD_GIT_DESCRIBE").unwrap_or_else(|| sentry::release_name!()),
```

Reply:
Not sure I can have a simple one liner since it is not a `String` but a `std::borrow::Cow` that is expected

Reply:
okaaay, gotcha